yew/src/html/scope.rs

```diff
@@ -253,40 +259,88 @@ impl<COMP: Component> Scope<COMP> {
     /// Creates a `Callback` which will send a batch of messages back
     /// to the linked component's update method when invoked.
     ///
+    /// The callback function's return type is generic to allow for dealing with both
+    /// Options and Vecs nicely. Option can be used when dealing with a callback that
```

Comment:
```suggestion
    /// `Option`s and `Vec`s nicely. An `Option` can be used when dealing with a callback that
```

Reply:
Oh man, you got the better of me again :)

I used the singular in my suggestion because it doesn't matter for the sentence and I think it looks a bit better.
What do you think?